lib/teleterm/clusters/cluster_servers.go

```diff
@@ -68,3 +71,75 @@ func (c *Cluster) GetServers(ctx context.Context) ([]Server, error) {
 
 	return results, nil
 }
+
+func (c *Cluster) GetServers(ctx context.Context, r *api.GetServersRequest) (*GetServersResponse, error) {
+	var (
+		clusterServers []types.Server
+		resp           *types.ListResourcesResponse
+		authClient     auth.ClientI
+		proxyClient    *client.ProxyClient
+		err            error
+	)
+
+	err = addMetadataToRetryableError(ctx, func() error {
```

Comment:
This function is closing the clients right after opening them.
Is this intended?
The defer statement is scoped to the inner (anonymous) function

Unless I'm missing something :bow: